### PR TITLE
Add phpunit:coverage command

### PR DESCRIPTION
**Pull request by weeman1337**
- `phpunit` command
- `phpunit:coverage` command with Igel config™
- editorconfig-checker ignore for that

Pipeline run https://chaos.expert/engelsystem/engelsystem/-/pipelines/37524
Locally the translation test failed :man_shrugging:

### CODE REVIEW

**Comment by MyIgel** [comment]
That variation of the editorconfig checker now also matches the `.idea` folder content (and thus also will probably a .code (or was it .vscode?) dir) which for example a run of `docker run -it --rm -v "$PWD":/check mstruebing/editorconfig-checker ec -v --exclude 'node_modules|vendor|public/(assets|coverage|.*\.xml)|.git'` will not do

**Comment by weeman1337** [comment]
Then I would say switch to a white list? You never know what files will appear for someone :man_shrugging:

**Comment by weeman1337** [comment]
:thinking: https://github.com/editorconfig-checker/editorconfig-checker/issues/99
Will check that in the afternoon

**Comment by MyIgel** [comment]
I rebased it to include the better fix for the ec exclude ;)

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,9 @@
             "composer phpcs:setup",
             "phpcbf -p"
         ],
-        "phpstan": "phpstan"
+        "phpstan": "phpstan",
+        "phpunit": "phpunit",
+        "phpunit:coverage": "phpunit --coverage-text --coverage-html public/coverage"
```
Comment (MyIgel) [inline composer.json:28]: At the moment that will only work if you have a local php installation with some existent coverage provider like `pcov` configured and enabled (like in the `.gitlab-ci.yml`) which is not the case in the workspace container at the moment.

Comment (weeman1337) [reply composer.json:28]: :upside_down_face:  https://github.com/engelsystem/engelsystem/blob/main/docker/dev/Dockerfile#L5-L7

**Comment by weeman1337** [comment]
Now also with documentation + [pcov is already enabled in the dev container](https://github.com/engelsystem/engelsystem/blob/main/docker/dev/Dockerfile#L5-L7)

File: composer.json (hunk 1 above)
Comment (MyIgel) [reply composer.json]: At least it was not *looksconfused

I mean: i explicitly tested it...

**Pull request merged**